storagenode/inspector/inspector.go

```diff
@@ -131,6 +141,7 @@ func (inspector *Endpoint) getDashboardData(ctx context.Context) (_ *pb.Dashboar
 		BootstrapAddress: strings.Join(bsNodes, ", "),
 		InternalAddress:  "",
 		ExternalAddress:  inspector.kademlia.Local().Address.Address,
+		DashboardAddress: inspector.consoleConfig.Address,
```

Comment:
This is not the address, this is the specification address... which could be `127.0.0.1:0`, you need the actual address.